Tests/Controller/app/config.yml

```diff
@@ -5,3 +5,7 @@ framework:
         engines: ['twig']
     test: ~
     assets: ~
+
+# This configuration can be removed once we go fully symfony ^4.0
+twig:
+    exception_controller:  twig.controller.exception::showAction
```

Comment:
Missing EOF. Check out if your edit doesn't add the new line at the end of file.